fix/DOCS-5466-Restructure ACUL Configure page (#1054)
auth0/docs-v2: main/docs/customize/login-pages/advanced-customizations/configure/reference.mdx

To use filtered mode, set `rendering_mode` to `advanced` and provide a `filters` object with a `match_type` (`includes_any` or `excludes_any`) and at least one entity array (`clients`, `organizations`, or `domains`). Each entity can be identified by `id` or by a `metadata` key/value pair.

## Head tags

Head tags are the primary mechanism for loading your custom UI into the ACUL host page. When Auth0 renders a screen in advanced mode, it serves a minimal HTML page and injects the elements defined in `head_tags` into the page `<head>`. These tags load the JavaScript and CSS asset bundles that make up your custom authentication screen.

Each entry in the `head_tags` array is an object with the following properties:

| **Property** | **Type** | **Description** |
| --- | --- | --- |
| `tag` | string | The HTML element type. Valid values: `script`, `link`, `style`, `meta`, `title`, `base`, `noscript`, `template`. |
| `attributes` | object | Attributes for the HTML element (for example, `src`, `href`, `defer`, `integrity`). Maximum of 10 attributes per tag. |
| `content` | string | Text or markup between the element's opening and closing tags. Maximum of 250 characters. |

> The `integrity` attribute is required for `script` and stylesheet tags. Auth0 uses [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to verify that assets have not been tampered with.

Reviewer: stylesheet link tags?

> The `integrity` attribute is required for `script` and stylesheet tags. Auth0 uses [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to verify that assets have not been tampered with.

Reviewer: ```suggestion
The `integrity` attribute is required for `script` and stylesheet tags. Auth0 uses [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to verify that there has been no modification to the assets.
```
active

```json
{
  "head_tags": [
    {
      "tag": "script",
      "attributes": {
        "src": "https://cdn.example.com/assets/{{ client.name }}/bundle.js",
        "defer": true,
        "integrity": "sha256-hashForClientA sha256-hashForClientB"
      }
    },
    {
      "tag": "link",
      "attributes": {
        "rel": "stylesheet",
        "href": "https://cdn.example.com/assets/{{ client.name }}/styles.css",
        "integrity": "sha256-cssHashForClientA sha256-cssHashForClientB"
      }
    }
  ]
}
```

You can also use head tags to load analytics scripts, custom fonts, or meta tags.

To replace the default Universal Login head tags (such as the favicon) with your own, set `default_head_tags_disabled` to `true`.

## Context data

Context data controls which tenant, client, organization, and branding information Auth0 sends to your custom screen at runtime. Auth0 delivers the data through the `universal_login_context` object on the host page. In your application code, use the [ACUL React SDK](/docs/libraries/acul/react-sdk) hooks or the [ACUL JS SDK](/docs/libraries/acul/js-sdk) methods to access context data.

> Each value you want available in your screen must be explicitly listed in the `context_configuration` array. Keys that resolve to empty or null values are removed from the payload.

Reviewer: ```suggestion
Each value you want available in your screen must be explicitly listed in the `context_configuration` array. Auth0 removes keys that resolve to empty or null values from the payload. 
```
active
